indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -138,3 +139,17 @@ class RHMenuEntries(RHEditingManagementBase):
     def _process(self):
         menu_entries = named_objects_from_signal(signals.menu.items.send('event-editing-sidemenu', event=self.event))
         return EditingMenuItemSchema(many=True).jsonify(menu_entries.values())
+
+
+class RHEditableTypes(RHEditingManagementBase):
+    def _process_GET(self):
+        return jsonify(editing_settings.get(self.event, 'editable_types').items())
+
+    @use_kwargs(EditableTypeArgs)
```

Comment:
No strong opinion, but since this schema is pretty simple you could also inline it:

```python
@use_kwargs({
    'editable_types': fields.List(EnumField(EditableType), required=True)
})
```